nautobot/core/settings.py

```diff
@@ -142,8 +142,14 @@
 #
 
 REST_FRAMEWORK_VERSION = VERSION.rsplit(".", 1)[0]  # Use major.minor as API version
+current_major, current_minor = REST_FRAMEWORK_VERSION.split(".")
+# We support all major.minor API versions from 1.2 to the present latest version.
+# This will need to be elaborated upon when we move to version 2.0
+assert current_major == "1", f"REST_FRAMEWORK_ALLOWED_VERSIONS needs to be updated to handle version {current_major}"
```

Comment:
My thinking on adding an assertion was that it will break quickly and obviously when we first do a version bump to 2.0.0-alpha.1, so as to guarantee that we don't forget to implement it. I don't mind encapsulating it in some way if you'd prefer but since this is basically a land-mine that I'm leaving in place intentionally, rather than something that we'll need to maintain long term, I didn't want to put in a lot of thought to it.

Reply:
Yeah that works for me.